float32[6] battery_level
int32[6] connection_status